addons/web/static/src/core/anchor_scroll_prevention.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module **/
+
+import { EventBus } from "@odoo/owl";
+import { browser } from "./browser/browser";
+
+export const ScrollerBus = new EventBus();
+
+browser.addEventListener("click", (ev) => {
+    const href = ev.target.closest("a")?.getAttribute("href");
```

Comment:
may be useful to be in capture mode.
An event stopped may trigger its default anyway, and this handler would not see it